CrossPlatformDesktopProject/CrossPlatformDesktopProject/NPC/Bat/Bat.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using CrossPlatformDesktopProject;
+using CrossPlatformDesktopProject.Link;
 using CrossPlatformDesktopProject.NPC;
 using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace CrossPlatformDesktopProject.NPC
 {
     class Bat : INpc
     {
-        private INpcState currentState;
+        public INpcState currentState;
```

Comment:
Does this need to be public? It would be best if we can reduce coupling and keep it private, but if it has to be public then it is fine to leave as is for now

Reply:
I think it has to remain public so that we can do things like `bat.currentState = new BatWalkSouth(bat);` in `BatWalkEast.cs`.

Reply:
Yes, for now anyways. I need it to be public so that it can go to the next cycle in the animation cycle. So for example within my `BatWalksEast` class, if the bat gets to a certain x position the `bat.CurrentState` will change to the 'BatWalksSouth' class. I need it to do that for sprint 2 just so that each enemy can show off multiple actions when cycled on to